cfgov/unprocessed/css/post.less

```diff
@@ -373,31 +373,6 @@
     - cfgov-post
 */
 
-.post_heading {
```

Comment:
Are we using these classes in any other places? https://github.com/cfpb/cfgov-refresh/search?utf8=%E2%9C%93&q=post_meta